server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -26,14 +27,17 @@ export async function runWithRetry<T>(
             logger?.info(`Error running ${callName}: retryCount ${retryCount}, error ${error}`);
             if (shouldIgnoreError !== undefined && shouldIgnoreError(error) === true) {
                 logger?.info(`Should ignore error for ${callName}`);
+                Lumberjack.info(`Should ignore error for ${callName}`);
                 break;
             } else if (shouldRetry !== undefined && shouldRetry(error) === false)
             {
                 logger?.info(`Should not retry ${callName}`);
+                Lumberjack.info(`Should not retry ${callName}`);
                 return Promise.reject(error);
             }
             if (retryCount >= maxRetries) {
                 logger?.info(`Error after retrying ${retryCount} times, rejecting`);
+                Lumberjack.info(`Error after retrying ${retryCount} times, rejecting`);
```

Comment:
Should this be `Lumberjack.error()` instead? Could also be the chance to change `logger?.info` to `logger?.error()`